data/human/ships.txt

```diff
@@ -2938,6 +2938,49 @@ ship "Scout"
 
 
 
+ship "Scrapper"
+	sprite "ship/scrapper"
+	thumbnail "thumbnail/scrapper"
+	attributes
+		category "Interceptor"
+		"cost" 60000
+		"shields" 450
+		"hull" 520
+		"required crew" 1
+		"bunks" 4
+		"mass" 62
+		"drag" 1.4
+		"heat dissipation" .8
+		"fuel capacity" 300
+		"cargo space" 21
+		"outfit space" 125
+		"weapon capacity" 12
+		"engine capacity" 58
+		weapon
+			"blast radius" 10
+			"shield damage" 80
+			"hull damage" 50
+			"hit force" 140
+	outfits
+		"Twin Blaster"
+		
+		"KP-6 Photovoltaic Panel" 4
+		"LP036a Battery Pack"
+		
+		"X2700 Ion Thruster"
+		"X2200 Ion Steering"
+		"Hyperdrive"
+		
+	engine -6 30
+	engine 6 30
+	gun 0 -31
+	leak "leak" 60 50
+	explode "tiny explosion" 10
+	explode "small explosion" 5
+	description "The Scrapper is less a model of ship, and more a general reference to ships cobbled together from the remains of scrapped shuttles. While Republic authorities prefer to dismiss them as at best an urban legend, and at worst an almost-illegal fabrication of unlicensed shipyards, these ships are where many aspiring captains from poorer planets get their start."
```

Comment:
```suggestion
	description "Rather than being a specific model of ship, the Scrappers are instead a family of vessels cobbled together from the remains of destroyed shuttles. On the rare occasion that Republic authorities bother to recognize their existence, Scrappers are treated as almost-illegal, patent-infringing death traps. Despite this, these ships are where many captains (and pirates) from poorer backgrounds get their start."
```

Reply:
I'm not too sure about `remains of destroyed shuttles`, feels too violent and specific..? I like the rest of the suggestion though.

Reply:
"decommissioned" sounds very military/official to me. I was thinking about it and maybe "remains of shuttle wrecks", but by definition it's pretty much the same as "destroyed", so maybe just "remains of shuttles" might work? Other words I was thinking of might be "plundered", "stolen", "broken down" which are kinda different but should give the same idea.

Reply:
Hmmm, how about merely "broken shuttles" ? I agree that decommissioned sounds rather formal, like ships that were intentionally declared out of service and dismantled properly; whereas I envision these as more being made from shuttles that wore out and just happened to end up parked in a junk spaceship lot. Nothing formal, they're just ships that just stopped getting fixed at some point, and then eventually people tore pieces out of them to stick together to make a new ship.

Some probably were destroyed, but that also sounds deliberate, whereas "broken" is just, well, it's the state they are in.